wit/guest.wit

```diff
@@ -1,10 +1,15 @@
-interface messaging-guest {
-    use messaging-types.{message, guest-configuration, error};
+interface incoming-handler {
+    use types.{message, error};
 
-    /// Returns the list of channels (and extension metadata within guest-configuration) that 
-    /// this component should subscribe to and be handled by the subsequent handler within guest-configuration
-    configure: func() -> result<guest-configuration, error>;
+    /// Whenever this guest receives a message in one of the subscribed topics, the message is
+    /// sent to this handler. The guest is responsible for matching on the channel and handling the
+    /// message accordingly. Implementors (such as hosts) calling this interface should make their
+    /// own decisions on how to handle errors returned from this function.
+    handle: func(ms: message) -> result<_, error>;
 
-    /// Whenever this guest receives a message in one of the subscribed channels, the message is sent to this handler
-    handler: func(ms: list<message>) -> result<_, error>;
+    // Subscribe to a list of topics (represented as `string`s) at runtime.
+    // Implementors should consider also allowing subscriptions to be made at compile time via
+    // some sort of configuration file. This function is intended to be called at the start of the
+    // guest's lifecycle before any messages are sent.
+    get-subscriptions: func() -> result<list<string>, error>;
```

Comment:
```suggestion
    // Returns to a list of topics (represented as `string`s) at runtime.
    // Implementors should consider also allowing subscriptions to be made at compile time via
    // some sort of configuration file. This function is intended to be called at the start of the
    // guest's lifecycle before any messages are sent.
    get-topics: func() -> result<list<string>, error>;
```

Subscription word her is confusing; the language has been established around `topic` as far as I can tell

Reply:
Addressed ✅